airflow/providers/snowflake/hooks/snowflake.py

```diff
@@ -179,6 +179,7 @@ def _get_conn_params(self) -> Dict[str, Optional[str]]:
             "role": self.role or role,
             "authenticator": self.authenticator or authenticator,
             "session_parameters": self.session_parameters or session_parameters,
+            "application": os.environ.get("AIRFLOW_SNOWFLAKE_PARTNER", "AIRFLOW")
```

Comment:
Can you add a little docs about this env variable to module docsteing?

Reply:
I will merge it as it is now - I want to really release providers. We can add it as follow-up.